src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -0,0 +1,70 @@
+var _ = require('underscore');
+var Model = require('../../core/model');
+
+/**
+ *  This model is used for getting the total amount of data
+ *  from the histogram widget (without any filter).
+ */
+
+module.exports = Model.extend({
+  defaults: {
+    url: '',
+    data: []
+  },
+
+  url: function () {
+    return this.get('url') + '?' + 'bins=' + this.get('bins');
+  },
+
+  initialize: function () {
+    this.bind('change:url change:bins', function () {
```

Comment:
Yes it is possible, if user updates dataview bins, this "unfilteredData" model should update its bins and get the data again.